app/templates/identity.hbs

```diff
@@ -41,21 +49,29 @@
                         {{on 'click' (fn this.handleCopy)}} />
 
                 {{else}}
-                    <Input
+                    <div class="btn-container">
+                        <Input
                         class='identity-Buttons'
                         type="button"
                         @value={{this.Chaincode}}
                         {{on 'click' (fn this.handleGenerateChaincode)}}
                         disabled={{this.generateChainCodeDisabled}}/>
+                        {{#if this.generateChainCodeDisabled}}
+                        <BsTooltip @title="Profile Url is empty !" @placement="right"/>
+                        {{/if}}
+                        
+                    </div>
```

Comment:
Can we please format this block of code well ?

Reply:
format the code and replaced the extra line spacing.